upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -256,7 +273,7 @@ function verify_mmhealth_summary() {
 	unhealthy=0
 	rm -f ${TEMP_MMHEALTH_FILE} >> /dev/null
 	nodename=$(oc get nodes |grep 'control'|grep 'Ready'|head -1|awk '{print $1}'|cut -d"." -f 1)
-	while read -r proc; do echo $proc >> ${TEMP_MMHEALTH_FILE}; done <<< "$(oc -n ibm-spectrum-scale rsh $nodename mmhealth cluster show|egrep 'NODE|GPFS|NETWORK|FILESYSTEM|DISK|FILESYSMGR|GUI|NATIVE_RAID|PERFMON|THRESHOLD')"
+	while read -r proc; do echo $proc >> ${TEMP_MMHEALTH_FILE}; done <<< "$(oc -n ibm-spectrum-scale rsh $nodename mmhealth cluster show|egrep 'NODE|GPFS|NETWORK|FILESYSTEM|DISK|FILESYSMGR|GUI|NATIVE_RAID|PERFMON|THRESHOLD|STRETCHCLUSTER')"
```

Comment:
just curious - is nodename a node or pod? I thought that 'oc rsh' worked with pods, not nodes and nodes would need regular ssh.